docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -87,6 +87,9 @@ <h4>{{ tab.title }}</h4>
     <tr uxd-api-property name="disabled" type="boolean" defaultValue="false">
         If <code>true</code> the tab can not be selected.
     </tr>
+    <tr uxd-api-property name="routerPath" type="string">
+        Define the tab router path
```

Comment:
I think this needs some clarification, at least a link to the Angular documentation for `routerLink` so that the format is understand, but also a description of what the effect of setting `routerPath` on a tab is.